README.md

```diff
@@ -22,6 +22,20 @@ You can configure which files to be analyzed in your `.codeclimate.yml` file.
 
 Other configurations can be made through a `.pylintrc` file. More information can be found on [Pylint's documentation](https://pylint.readthedocs.io/en/latest/)
 
+PYTHONPATH can be modified by setting the PYTHONPATH configuration variable:
+
+```yaml
+engines:
+  pylint:
+    enabled: true
+    channel: "beta"
+    config:
```

Comment:
I think this extra `config` can be confusing, can you please put `PYTHONPATH` directly under `pylint`?

Reply:
OK, I did this because I wasn't sure if code climate would support it. Let me try.